config/application.rb

```diff
@@ -3,7 +3,7 @@
 require "rails"
 require "rails/all"
 require "active_storage/engine"
-require_relative "../lib/rack_x_robots_tag"
+require_relative "../lib/rack/x_robots_tag"
```

Comment:
@jamesjefferies Something to do with autoloading doesn't like that the file name wasn't related to the class name and meant test kept cycling and wouldn't load.

Reply:
woah, that's a bit weird isn't it? I didn't even look to make sure the test deploy had worked - more fool me... you fixed it then?

Reply:
thanks :)

Reply:
Yes, this seems to fix it!